tests/test_version_command.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+
+from onebuild.main import run
+from onebuild import __version__
+
+VERSION_MESSAGE = "1build {version}".format(version=__version__)
```

Comment:
Do not use the same logic as source code to check string.

rather write an exact string that you want. 

Test should fail if version did not march.

Reply:
updated the code.

added negative test as well, this will fail if `VERSION` is not matching.

Reply:
👍